docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,262 @@
+# KubeStellar Hub chart usage
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choice of user-defined KubeFlex Control Planes (CPs).
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.2
+```
+
+## Pre-requisites
+
+To install the Helm chart the only requirement is [Helm](https://helm.sh/).
+However, additional executables may be required to create/manage the cluster(s) (_e.g._, Kind and kubectl),
+to join Workload Execution Clusters (WEKs) (_e.g._, clusteradm),
+and to interact with Control Planes (_e.g._, kubectl), _etc_.
+For such purpose, a full list of executable that may be required can be fund [here](./pre-reqs.md).
+
+The setup of KubStellar via the Hub chart requires the existance of at least one hosting cluster.

```

Comment:
The setup does not use multiple hosting clusters. I would say "requires the existence of a KubeFlex hosting cluster".

Reply:
ok